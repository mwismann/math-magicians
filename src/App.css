.App {
  height: 100vh;
  width: 100%;
  background-color: bisque;
}
