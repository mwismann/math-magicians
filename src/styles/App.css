.App {
  width: 100%;
  background-color: rgb(39, 35, 43);
}
