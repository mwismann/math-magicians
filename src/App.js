// eslint-disable-next-line import/no-extraneous-dependencies
import { Routes, Route } from 'react-router-dom';
import Header from './components/Header';
import Home from './components/Home';
import RenderCalculator from './components/Calculator';
import FetchQuote from './components/Fetch';
import './styles/App.css';

function App() {
  return (
    <>
      <Header />
      <Routes>
        <Route path="/" element={<Home />} />
        <Route path="/calculator" element={<RenderCalculator />} />
        <Route path="/quote" element={<FetchQuote />} />
      </Routes>
    </>
  );
}

export default App;
