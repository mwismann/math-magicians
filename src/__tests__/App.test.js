import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import App from '../App';

test('renders App', () => {
  const text = renderer.create(<BrowserRouter><App /></BrowserRouter>);
  expect(text).toMatchSnapshot();
});
