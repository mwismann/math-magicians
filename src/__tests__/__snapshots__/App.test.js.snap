// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App 1`] = `
Array [
  <header>
    <div
      className="logo-container"
    >
      <h1>
        Math Magicians
      </h1>
    </div>
    <nav>
      <ul>
        <li>
          <a
            aria-current="page"
            className="active"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
         | 
        <li>
          <a
            className=""
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
         | 
        <li>
          <a
            className=""
            href="/quote"
            onClick={[Function]}
          >
            Quote
          </a>
        </li>
      </ul>
    </nav>
  </header>,
  <div
    className="App"
  >
    <div
      className="home-wrapper"
    >
      <h2>
        Welcome to our page!
      </h2>
      <section
        className="home-about"
      >
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
        </p>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
        </p>
      </section>
    </div>
  </div>,
]
`;
